### ISSUE TITLE
Afegir inclinació a les pantalles

### ISSUE THREAD

**Issue by fpraderio** [fpraderio/castelloscopi#3]
Ara mateix les pantalles són verticals, i això fa més dificil veure les animacions.

Es podrien inclinar totes una mica capa el centre.

**Comment by PraderioM** [comment]
Castelloscopi non legge rotazioi rispetto all'asse z que sono
imprescindibili per fare bene la inclinazione.
Ho fatto un programma que indica cuanto deve essere inclinato ognuno dei 43
schermi rispetto ad ogni asse.
lo puoi trovare quí
https://github.com/PraderioM/rotacio3d

2016-01-22 23:58 GMT+01:00 Francesco notifications@github.com:

**Comment by fpraderio** [comment]
Encara no he tingut temps de provar https://github.com/PraderioM/rotacio3d . 
Entenc que castellers.json, és el resultat d'aplicar el canvi d'angle. És així?

**Comment by PraderioM** [comment]
no, la rotació sobre l'eix x i z de cada screen es mostra per pantalla una
vegada executat el html.
Encara ho haig d'arreclar perquè el resultat es mostri com dius tu.

El dia 31 de gener de 2016, 17:58, Francesco notifications@github.com ha
escrit:

**Comment by PraderioM** [comment]
fet ara l'arxiu castellers.json és el resultat d'aplicar el canvi.

El dia 31 de gener de 2016, 22:01, Marco Praderio Bova praderiom@gmail.com
ha escrit:
